doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -24,27 +23,36 @@ Each set of ephemerides predictions is valid for a specific four-hour period wit
 A device using P-GPS downloads the ephemeris predictions from the cloud, stores them in its flash memory, and later injects them into the GNSS module when needed.
 
 P-GPS is designed for devices that are frequently disconnected from the cloud but need periodic GNSS fixes as quickly as possible to save power.
-This is possible because a device can download the broadcasted information and predictions of satellite data provided through P-GPS (or also A-GPS) at a faster rate from nRF Cloud than from the data links of the satellites.
+This is possible because a device does not need to download ephemerides from the satellite broadcast.
 However, P-GPS should not be used for general use cases that already work with :term:`Assisted GPS (A-GPS)` only.
 
 .. note::
    When using two-week ephemeris prediction sets, the TTFF towards the end of the second week increases due to the accumulated errors in the predictions and the decreases in the number of satellite ephemerides in the later prediction periods.
 
-P-GPS requires a cloud connection approximately once a week, depending on the configuration settings.
-A-GPS assistance data expires after two hours.
-A-GPS downloads new assistance data from the cloud to replace expired data when your application requests a position fix.
+P-GPS requires a cloud connection approximately once a week to download new predictions, depending on the configuration settings.
+With A-GPS, new ephemerides are needed on average every two hours, or if the fix interval is longer, whenever GNSS is started.
+
+.. note::
+   To use the nRF Cloud P-GPS service, you need an nRF Cloud account, and the device needs to be associated with the account.
```

Comment:
can you make the wording the same for the A-GPS note? (nrf_cloud_agps.rst line 22)

Reply:
Sure.